.github/workflows/selfserve-generate.yaml

```diff
@@ -0,0 +1,65 @@
+name: Generate SDK
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: 'SDK Name'
+        required: true
+        type: string
+      version:
+        description: 'SDK Version'
+        required: true
+        type: string
+      templates:
+        description: 'Path to the templates directory'
```

Comment:
Can we add a default value to the templates directory based on the structure that we currently follow?

Reply:
Discussed with Noor, and decided to put on hold till we have merged templates